web/src/app/(protected)/contacts/content.tsx

```diff
@@ -0,0 +1,36 @@
+'use client'
+
+import { Row } from '@/lib/styled/global'
+import { useState } from 'react'
+import { ChatsSection } from './chatSection'
+import { ListSection } from './listSection'
+
+export const Content = () => {
+  const [openSelectorChats, setOpenSelectorChats] = useState<string | null>(
+    null,
+  )
+  const [openSelectorLists, setOpenSelectorLists] = useState<string | null>(
+    null,
+  )
+
+  const handleOpenSelectorChats = (name: string) => {
+    setOpenSelectorChats((prev) => (prev === name ? null : name))
+  }
+
+  const handleOpenSelectorLists = (name: string) => {
+    setOpenSelectorLists((prev) => (prev === name ? null : name))
+  }
+
```

Comment:
```suggestion
  const [openSelectorChats, setOpenSelectorChats] = useState<string | null>(
    null,
  )
  const [openSelectorLists, setOpenSelectorLists] = useState<string | null>(
    null,
  )

  const handleOpenSelectorChats = (name: string) => {
    setOpenSelectorChats((prev) => (prev === name ? null : name))
  }

  const handleOpenSelectorLists = (name: string) => {
    setOpenSelectorLists((prev) => (prev === name ? null : name))
  }

```

Isso deveria estar dentro do ChatsSection e dentro do ListSection